### PR TITLE
Move response status to views

### PR DESCRIPTION
**Pull request by LJBabbage**
This branch is based of https://github.com/ONSdigital/ras-collection-instrument/pull/50, which needs to be merged first. It moves the response status logic to the views rather then being in the controller

### CODE REVIEW

File: application/controllers/survey_response.py [hunk 1]
```diff
@@ -6,82 +6,37 @@
...
     The survey response from a respondent
     """
-    def add_survey_response(self, case_id, file):
+    def add_survey_response(self, case_id, file, file_name):
         """
         Encrypt and upload survey response to rabbitmq
         :param case_id: A case id
         :param file: A file object from which we can read the file contents
+        :param file_name: The filename
         :return: Returns status code and message
```
Comment (benjefferies) [inline application/controllers/survey_response.py]: Doc needs updating

File: application/controllers/survey_response.py [hunk 2]
```diff
@@ -6,82 +6,37 @@
...
+FILE_EXTENSION_ERROR = 'The spreadsheet must be in .xls or .xlsx format'
 FILE_NAME_LENGTH_ERROR = 'The file name of your spreadsheet must be less than 50 characters long'
-UPLOAD_SUCCESSFUL = 'Upload successful'
-UPLOAD_UNSUCCESSFUL = 'Upload failed'
-INVALID_UPLOAD = 'The upload must have valid case_id and a file attached'
-NOT_FOUND_SERVICE_RESPONSE = 'A request to an external service returned no data'
 QUEUE_NAME = 'Seft.Responses'
 
 
 class SurveyResponse(object):
```
Comment (benjefferies) [inline application/controllers/survey_response.py]: This probably doesn't need to be a class and could just be a few functions

File: tests/controllers/test_collection_instrument.py [hunk 3]
```diff
@@ -64,25 +61,14 @@ def test_get_instrument_by_search_string_invalid_search_value(self):
         with self.assertRaises(RasDatabaseError):
             self.collection_instrument.get_instrument_by_search_string('{\"COLLECTION_EXERCISE\": \"invalid_uuid\"}')
 
-    def test_get_instrument_by_id(self):
```
Comment (benjefferies) [inline tests/controllers/test_collection_instrument.py]: Do we need a test for get_instrument_json_by_id?

Comment (LJBabbage) [reply tests/controllers/test_collection_instrument.py]: The view test covers it

File: application/controllers/survey_response.py (hunk 2 above)
Comment (LJBabbage) [reply application/controllers/survey_response.py]: Yep, but maybe one for the future. Just need to get this PR in before it disintegrates. Its been alive far to long

**Pull request merged**